util/envDynawo.sh

```diff
@@ -1235,12 +1300,8 @@ case $MODE in
     build_3rd_party_version || error_exit "Error while building 3rd party version"
     ;;
 
-  compileModelicaOMCHelp)
```

Comment:
Why is this removed?

Reply:
No need for this as compilerModelica --help does the same, I think that we have to much options and with the autocompletion --help will appear in the suggestions